scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -126,35 +126,38 @@ def test__strategy_resolves(self):
     def test__mutate1(self):
         # strategies */1/*, i.e. rand/1/bin, best/1/exp, etc.
         result = np.array([0.05])
-        trial = self.dummy_solver2._best1((2, 3, 4, 5, 6))
+        trial = self.dummy_solver2._best1(np.array([2, 3, 4, 5, 6]))
```

Comment:
I see a list needs to be changed to an array. Any chance this changes the public API? (Doesn't look like it, but I thought I should ask.)

Reply:
No it doesn't, everything is private. I needed to convert to an array here because the method tries to take the transpose of it.